src/components/InputTodo.js

```diff
@@ -0,0 +1,45 @@
+import React, { useRef } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+import '../css/inputTodo.css';
+
+function InputTodo({ addItem, list, id }) {
+  const ref = useRef();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newList = [
+      {
+        id,
+        title: ref.current.value,
+        done: false,
+      },
+    ];
+    const add = list.concat(newList);
+    addItem(add);
+
+    ref.current.value = '';
+  };
+
+  return (
+    <div className="input">
+      <form onSubmit={handleSubmit}>
+        <input type="text" ref={ref} />
```

Comment:
- Please consider using useState in place of useRef. It is considered general best practice the use useState when the UI needs to be re-rendered and useRef when no re-render is required. read more about this [here](https://dev.to/trinityyi/when-to-use-useref-instead-of-usestate-3h4o#:~:text=A%20rule%20of%20thumb%20is,component%20when%20the%20state%20changes.)

Reply:
Thank you will do that